optuna/visualization/_terminator_improvement.py

```diff
@@ -40,6 +40,54 @@ def plot_terminator_improvement(
     error_evaluator: BaseErrorEvaluator | None = None,
     min_n_trials: int = DEFAULT_MIN_N_TRIALS,
 ) -> "go.Figure":
+    """Plot the potentials for future objective improvement.
+
+    This function visualizes the objective improvement potentials, evaluated
+    with `improvement_evaluator`.
+    It helps to determine whether we should continue the optimization or not.
+    You can also plot the error evaluated with
+    `error_evaluator` if the `plot_error` argument is set to True.
+    Note that this function may take some time to compute
+    the improvement potentials.
+
+    Example:
+
+        The following code snippet shows how to plot improvement potentials.
+
+        .. plotly::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", -100, 100)
+                y = trial.suggest_categorical("y", [-1, 0, 1])
+                return x ** 2 + y
+
+            sampler = optuna.samplers.TPESampler(seed=10)
+            study = optuna.create_study(sampler=sampler)
+            study.optimize(objective, n_trials=10)
```

Comment:
:nit: I think more `n_trials` is needed to show the effect of `min_n_trials`. (i.e. 30). What do you think?